### ISSUE TITLE
add purpose and notes fields to mover

### ISSUE THREAD

**Issue by stufraser1** [GFDRR/rdl-data#20]
Hazard/ged4all/loss.contribution.purpose
Hazard/ged4all/loss.contribution.notes

Does not exist in mover - improve consistency by adding this

**Comment by stufraser1** [comment]
Could be done by adding additional Contribution table to MOVER. @ConnectedSystems can it be done in SQL alchemy version?

**Comment by matamadio** [comment]
Added to docs: mover.contribution like the other ones.
Should we make "contribution" a global table that each sub-set can relate to?

**Comment by ConnectedSystems** [comment]
If `Contribution` is intended to hold the same data, then it should be part of the `common` schema.

~Note that the `f_core` (which I am taking to be the `Contribution` table for the Vulnerability/Mover schema) has additional, possibly context-specific fields.~

I've added `notes`, `version` and `purpose` fields for now but it would be good for this to be standardized.

EDIT: I see now that a separate contribution table has been added to Vulnerability. 
@MamadioCMCC  and I will do a review to remove any possible duplicates between the two.
